[2023-08-29T19:43:56.303+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-29T19:43:56.313+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-29T19:43:56.323+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-29T19:43:56.325+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-29T19:43:56.342+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-24 00:00:00+00:00
[2023-08-29T19:43:56.349+0000] {standard_task_runner.py:57} INFO - Started process 353 to run task
[2023-08-29T19:43:56.354+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-24T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpcp2jallu']
[2023-08-29T19:43:56.357+0000] {standard_task_runner.py:85} INFO - Job 25: Subtask import_data_to_csv
[2023-08-29T19:43:56.392+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-29T19:43:56.490+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [running]> on host e69093e17de8
[2023-08-29T19:43:56.522+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-29T19:43:56.628+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-24T00:00:00+00:00'
[2023-08-29T19:43:56.634+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-29T19:43:56.636+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /Users/jinee.hira/Dropbox/virtual_envs/austin-flow/***/dags/data_ingestion.py --date 2023-08-24']
[2023-08-29T19:43:56.659+0000] {subprocess.py:86} INFO - Output:
[2023-08-29T19:43:56.815+0000] {subprocess.py:93} INFO - python: can't open file '/Users/jinee.hira/Dropbox/virtual_envs/austin-flow/***/dags/data_ingestion.py': [Errno 2] No such file or directory
[2023-08-29T19:43:56.823+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-08-29T19:43:56.845+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-08-29T19:43:56.857+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230824T000000, start_date=20230829T194356, end_date=20230829T194356
[2023-08-29T19:43:56.876+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 25 for task import_data_to_csv (Bash command failed. The command returned a non-zero exit code 2.; 353)
[2023-08-29T19:43:56.937+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-08-29T19:43:56.971+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-30T18:56:46.296+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:46.312+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T18:56:46.331+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T18:56:46.333+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T18:56:46.360+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-24 00:00:00+00:00
[2023-08-30T18:56:46.365+0000] {standard_task_runner.py:57} INFO - Started process 551 to run task
[2023-08-30T18:56:46.377+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-24T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmplxtv99r2']
[2023-08-30T18:56:46.381+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask import_data_to_csv
[2023-08-30T18:56:46.411+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:46.476+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [running]> on host 5e7fbf8b7901
[2023-08-30T18:56:46.498+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:46.610+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-24T00:00:00+00:00'
[2023-08-30T18:56:46.613+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T18:56:46.616+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /Users/jinee.hira/Dropbox/virtual_envs/austin-flow/***/dags/data_ingestion.py --date 2023-08-24']
[2023-08-30T18:56:46.629+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T18:56:46.706+0000] {subprocess.py:93} INFO - python: can't open file '/Users/jinee.hira/Dropbox/virtual_envs/austin-flow/***/dags/data_ingestion.py': [Errno 2] No such file or directory
[2023-08-30T18:56:46.710+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-08-30T18:56:46.727+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-08-30T18:56:46.732+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230824T000000, start_date=20230830T185646, end_date=20230830T185646
[2023-08-30T18:56:46.756+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 11 for task import_data_to_csv (Bash command failed. The command returned a non-zero exit code 2.; 551)
[2023-08-30T18:56:46.785+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-08-30T18:56:46.809+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:20:00.549+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:20:00.560+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T19:20:00.572+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T19:20:00.573+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:20:00.588+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-24 00:00:00+00:00
[2023-08-30T19:20:00.593+0000] {standard_task_runner.py:57} INFO - Started process 268 to run task
[2023-08-30T19:20:00.601+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-24T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpbk7831go']
[2023-08-30T19:20:00.605+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask import_data_to_csv
[2023-08-30T19:20:00.630+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:20:00.685+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [running]> on host 884d86529f65
[2023-08-30T19:20:00.713+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:20:00.820+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-24T00:00:00+00:00'
[2023-08-30T19:20:00.822+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:20:00.824+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python ./dags/data_ingestion.py --date 2023-08-24']
[2023-08-30T19:20:00.844+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:20:00.942+0000] {subprocess.py:93} INFO - python: can't open file './dags/data_ingestion.py': [Errno 2] No such file or directory
[2023-08-30T19:20:00.949+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-08-30T19:20:00.965+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-08-30T19:20:00.972+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230824T000000, start_date=20230830T192000, end_date=20230830T192000
[2023-08-30T19:20:00.993+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 9 for task import_data_to_csv (Bash command failed. The command returned a non-zero exit code 2.; 268)
[2023-08-30T19:20:01.054+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-08-30T19:20:01.090+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:33:39.611+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:33:39.620+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T19:33:39.629+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T19:33:39.631+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:33:39.638+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-24 00:00:00+00:00
[2023-08-30T19:33:39.643+0000] {standard_task_runner.py:57} INFO - Started process 392 to run task
[2023-08-30T19:33:39.645+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-24T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp7w8f6edm']
[2023-08-30T19:33:39.649+0000] {standard_task_runner.py:85} INFO - Job 19: Subtask import_data_to_csv
[2023-08-30T19:33:39.661+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:33:39.699+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [running]> on host f6bcf103ff79
[2023-08-30T19:33:39.714+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:33:39.767+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-24T00:00:00+00:00'
[2023-08-30T19:33:39.769+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:33:39.771+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-24']
[2023-08-30T19:33:39.783+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:33:41.713+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:33:41.737+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230824T000000, start_date=20230830T193339, end_date=20230830T193341
[2023-08-30T19:33:41.802+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:33:41.819+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:41:53.566+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:53.582+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T19:41:53.592+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T19:41:53.595+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:41:53.614+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-24 00:00:00+00:00
[2023-08-30T19:41:53.619+0000] {standard_task_runner.py:57} INFO - Started process 328 to run task
[2023-08-30T19:41:53.631+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-24T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpatwgwx0p']
[2023-08-30T19:41:53.636+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask import_data_to_csv
[2023-08-30T19:41:53.656+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:53.705+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [running]> on host 81eeb4b7c8a9
[2023-08-30T19:41:53.731+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:53.817+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-24T00:00:00+00:00'
[2023-08-30T19:41:53.822+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:41:53.828+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-24']
[2023-08-30T19:41:53.854+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:42:00.298+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:42:00.335+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230824T000000, start_date=20230830T194153, end_date=20230830T194200
[2023-08-30T19:42:00.397+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:42:00.421+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:50:53.717+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:53.731+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T19:50:53.741+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T19:50:53.742+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:50:53.760+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-24 00:00:00+00:00
[2023-08-30T19:50:53.767+0000] {standard_task_runner.py:57} INFO - Started process 303 to run task
[2023-08-30T19:50:53.774+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-24T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpaq2c1s1j']
[2023-08-30T19:50:53.778+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask import_data_to_csv
[2023-08-30T19:50:53.806+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:53.875+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [running]> on host 206ecb1dc8df
[2023-08-30T19:50:53.902+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:53.988+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-24T00:00:00+00:00'
[2023-08-30T19:50:53.992+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:50:53.994+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-24']
[2023-08-30T19:50:54.008+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:51:01.757+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:51:01.826+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230824T000000, start_date=20230830T195053, end_date=20230830T195101
[2023-08-30T19:51:01.919+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:51:01.952+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:54:26.259+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:26.270+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T19:54:26.283+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T19:54:26.284+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:54:26.298+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-24 00:00:00+00:00
[2023-08-30T19:54:26.303+0000] {standard_task_runner.py:57} INFO - Started process 260 to run task
[2023-08-30T19:54:26.315+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-24T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpi44wh0zj']
[2023-08-30T19:54:26.323+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask import_data_to_csv
[2023-08-30T19:54:26.359+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:26.421+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [running]> on host 86f9e97d6c70
[2023-08-30T19:54:26.435+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:26.518+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-24T00:00:00+00:00'
[2023-08-30T19:54:26.520+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:54:26.522+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-24']
[2023-08-30T19:54:26.551+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:54:30.186+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:54:30.243+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230824T000000, start_date=20230830T195426, end_date=20230830T195430
[2023-08-30T19:54:30.296+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:54:30.329+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:05:49.235+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:49.245+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T20:05:49.260+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T20:05:49.261+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:05:49.272+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-24 00:00:00+00:00
[2023-08-30T20:05:49.278+0000] {standard_task_runner.py:57} INFO - Started process 464 to run task
[2023-08-30T20:05:49.285+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-24T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpn2ms65cs']
[2023-08-30T20:05:49.292+0000] {standard_task_runner.py:85} INFO - Job 17: Subtask import_data_to_csv
[2023-08-30T20:05:49.317+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:49.378+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [running]> on host 3938d7abd9ff
[2023-08-30T20:05:49.404+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:49.492+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-24T00:00:00+00:00'
[2023-08-30T20:05:49.496+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:05:49.497+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-24']
[2023-08-30T20:05:49.509+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:05:56.541+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:05:56.624+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230824T000000, start_date=20230830T200549, end_date=20230830T200556
[2023-08-30T20:05:56.679+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:05:56.736+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:33:50.108+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:50.123+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T20:33:50.140+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T20:33:50.142+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:33:50.164+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-24 00:00:00+00:00
[2023-08-30T20:33:50.169+0000] {standard_task_runner.py:57} INFO - Started process 285 to run task
[2023-08-30T20:33:50.185+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-24T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpz7xc0iar']
[2023-08-30T20:33:50.189+0000] {standard_task_runner.py:85} INFO - Job 10: Subtask import_data_to_csv
[2023-08-30T20:33:50.220+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:50.289+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [running]> on host 961f38e8363e
[2023-08-30T20:33:50.313+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:50.395+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-24T00:00:00+00:00'
[2023-08-30T20:33:50.398+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:33:50.402+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-24']
[2023-08-30T20:33:50.413+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:33:56.195+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:33:56.265+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230824T000000, start_date=20230830T203350, end_date=20230830T203356
[2023-08-30T20:33:56.315+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:33:56.335+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:48:23.526+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:23.539+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T20:48:23.552+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T20:48:23.555+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:48:23.580+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-24 00:00:00+00:00
[2023-08-30T20:48:23.601+0000] {standard_task_runner.py:57} INFO - Started process 343 to run task
[2023-08-30T20:48:23.611+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-24T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp0pjmhy_5']
[2023-08-30T20:48:23.615+0000] {standard_task_runner.py:85} INFO - Job 14: Subtask import_data_to_csv
[2023-08-30T20:48:23.640+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:23.697+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [running]> on host 8f66711a2898
[2023-08-30T20:48:23.721+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:23.801+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-24T00:00:00+00:00'
[2023-08-30T20:48:23.804+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:48:23.805+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-24']
[2023-08-30T20:48:23.843+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:48:30.687+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:48:30.727+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230824T000000, start_date=20230830T204823, end_date=20230830T204830
[2023-08-30T20:48:30.788+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:48:30.818+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T21:07:36.651+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:36.662+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T21:07:36.670+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T21:07:36.672+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T21:07:36.692+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-24 00:00:00+00:00
[2023-08-30T21:07:36.699+0000] {standard_task_runner.py:57} INFO - Started process 295 to run task
[2023-08-30T21:07:36.706+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-24T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpm_luehpe']
[2023-08-30T21:07:36.710+0000] {standard_task_runner.py:85} INFO - Job 12: Subtask import_data_to_csv
[2023-08-30T21:07:36.738+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:36.855+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [running]> on host cbc745b7550b
[2023-08-30T21:07:36.937+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:37.057+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-24T00:00:00+00:00'
[2023-08-30T21:07:37.074+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T21:07:37.076+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-24']
[2023-08-30T21:07:37.106+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T21:07:52.776+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T21:07:52.862+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230824T000000, start_date=20230830T210736, end_date=20230830T210752
[2023-08-30T21:07:52.939+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T21:07:52.978+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T21:17:04.214+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:17:04.226+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T21:17:04.241+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [queued]>
[2023-08-30T21:17:04.243+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T21:17:04.265+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-24 00:00:00+00:00
[2023-08-30T21:17:04.276+0000] {standard_task_runner.py:57} INFO - Started process 354 to run task
[2023-08-30T21:17:04.290+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-24T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpl3m4av84']
[2023-08-30T21:17:04.294+0000] {standard_task_runner.py:85} INFO - Job 13: Subtask import_data_to_csv
[2023-08-30T21:17:04.318+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:17:04.373+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-24T00:00:00+00:00 [running]> on host 5b9f14bf20e0
[2023-08-30T21:17:04.396+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:17:04.486+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-24T00:00:00+00:00'
[2023-08-30T21:17:04.490+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T21:17:04.493+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-24']
[2023-08-30T21:17:04.506+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T21:17:14.613+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T21:17:14.669+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230824T000000, start_date=20230830T211704, end_date=20230830T211714
[2023-08-30T21:17:14.705+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T21:17:14.727+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
